packages/sources/coinmarketcap/src/endpoint/globalMarketCap.ts

```diff
@@ -27,6 +27,7 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   response.data.result = Requester.validateResultNumber(response.data, [
     'data',
     'quote',
+    'quote',
```

Comment:
Wanted to double check this.  I had to add this to get the adapter to work.  Is there an existing bug with this adapter?

Reply:
It's working for me from the current `develop` branch :thinking:

Reply:
I must have been looking at another endpoint's response.  Confirmed that this change is not needed.